.github/workflows/markdown-lint.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: markdown-lint
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I don't think we need this line. I don't use it on protractor-helper, for instance (see .https://github.com/wlsf82/protractor-helper/blob/master/.github/workflows/continuous-integration.yml#L6)

Reply:
done